doozer/doozerlib/cli/scan_sources_konflux.py

```
@@ -106,6 +107,23 @@ def __init__(
+        if image_meta.distgit_key in self.skipped_image_names:
+            return
+        self.skipped_image_names.add(image_meta.distgit_key)
+        self.logger.warning("[%s] %s", image_meta.distgit_key, issue)
+        self.issues.append({'name': image_meta.distgit_key, 'issue': issue})
+
+    def _prepare_image_for_scan(self, image_meta: ImageMetadata, phase: str) -> bool:
+        if image_meta.distgit_key in self.skipped_image_names:
+            return False
+        try:
+            image_meta.ensure_canonical_builders_resolved()
+        except Exception as err:
```

Ruff (0.15.11)

[warning] Do not catch blind exception: `Exception`

(BLE001)